### ISSUE TITLE
Nonetype error with the latest update

### ISSUE THREAD

**Issue by angelblue05** [BigNoid/script.skinshortcuts#10]
I'm using Kodi 14.2 with Aeon Nox

``` python
12:34:27 T:42268  NOTICE: -->Python Interpreter Initialized<--
12:34:28 T:42268  NOTICE: Previous line repeats 1 times.
12:34:28 T:42268   ERROR: Traceback (most recent call last):
12:34:28 T:42268   ERROR:   File "C:\Users\Angel\AppData\Roaming\Kodi\addons\script.skinshortcuts\resources\lib\xmlfunctions.py", line 87, in buildMenu
12:34:28 T:42268   ERROR:     self.writexml( profilelist, mainmenuID, groups, numLevels, buildMode, progress, options )
12:34:28 T:42268   ERROR:   File "C:\Users\Angel\AppData\Roaming\Kodi\addons\script.skinshortcuts\resources\lib\xmlfunctions.py", line 406, in writexml
12:34:28 T:42268   ERROR:     submenudata = DATA._get_shortcuts( submenu, submenuDefaultID, True, profile[0] )
12:34:28 T:42268   ERROR:   File "C:\Users\Angel\AppData\Roaming\Kodi\addons\script.skinshortcuts\resources\lib\datafunctions.py", line 167, in _get_shortcuts
12:34:28 T:42268   ERROR:     self._process_shortcuts( tree, group, profileDir )
12:34:28 T:42268   ERROR:   File "C:\Users\Angel\AppData\Roaming\Kodi\addons\script.skinshortcuts\resources\lib\datafunctions.py", line 205, in _process_shortcuts
12:34:28 T:42268   ERROR:     allGroupOverrides = skinoverrides.findall( "groupoverride" )
12:34:28 T:42268   ERROR: AttributeError: 'NoneType' object has no attribute 'findall'
```

**Comment by marcelveldt** [comment]
I'll check it out...

**Comment by angelblue05** [comment]
hey Marcel! I didn't know you were working on this!

**Comment by marcelveldt** [comment]
hey!, yeah, since Unfledged (Original author of skinshortcuts) left the Kodi community, Bignoid and phil65 took over the maintenance of the script. Because I utilize every bit of the script in my skin I sometimes fix a few issues too  ;-)

A few days back I fixed some issue in preparation for Isengard in the skin and placed a testversion of the script on our beta repo in order to let some users testdrive the changes. That's why you got the update ;-) Luckily this ironed out a single issue I didn't think of. It's now fixed and submitted to Git.

@BigNoid , for you to review I've submitted a PR with the fix for this issue and some other small issue a user on the forums came up with.

This issue can be closed once the PR is done.

@angelblue05, I'll update the testversion on our mediabrowser repo for you..
